int32 command
int32 hand
float32 value
---
bool success